service/resource/deployment/resource.go

```diff
@@ -57,6 +58,7 @@ func DefaultConfig() Config {
 	}
 }
 
+// Resource manages Azure ARM templates deployments.
 type Resource struct {
```

Comment:
I started to use automated linter and this is a bitch :) So expect me to add some godoc.

Reply:
IMO we started to remove these comments since they are quite useless and everywhere the same. Should we keep them though?

Reply:
OK, I'll remove. I thought for generic names like this it makes sense. But probably I end up adding comments like `getCerts returns a cert.`.

Reply:
Done.